src/scss/components/_checkbox.scss

```diff
@@ -15,7 +15,7 @@ $checkbox-label-vertical-padding: 0.75rem;
       center;
     background-color: $white;
     background-size: 0;
-    border: 2px solid $input-color;
+    border: 2px solid $input-border;
```

Comment:
Sorry, my previous comment wasn't clear (my bad! 🤦)

I was thinking more along the lines of:
`border: $input-border;`
where 
`$input-border: 2px solid $black;` or 
`$input-border: 2px solid $input-border-color;`

Reply:
Ah that makes sense to me now. I'll add `input-border-color` to `colors` and I think I will create a new `_forms.scss` in `scss/variables` which includes the full border property. I'm not entirely sure of the abstraction of calling it forms, though. Let me know your thoughts